### ISSUE TITLE
Document Fully Noded Integration ( Expose Bitcoin RPC via Tor v3)

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver-doc#364]
Finally, it's here! 🍾 
We need to show users how to connect their BTCPay to their fully noded.

@britttttk any ideas where we can add the instructions on how to connect to fully noded?

@Fonta1n3 may be interested in documenting this one.

**Comment by pavlenex** [comment]
From the blog-post today. Probably needs a bit more `documentation`touch to it, but in general t's quite easy.

```
To connect your BTCPay node to Fully Noded:

    1. If you’re iOS user, download Fully Noded from the App store.
    2. In BTCPay, go to Server Settings > Services and click on the Full Node RPC.
    3. Open your Fully noded app, Settings > Node Manager, Add new node.
    4. Enable Tor in Fully noded and can a QR code displayed on your BTCPay.
    5. Your Bitcoin full node is now connected to Fully Noded.
```

**Comment by maltokyo** [comment]
I did this without any "training material" ;)  The perfect integration.
(Joking, documentation is always good).

I still do not understand really the need to control the full node from iOS, because we do not even have wallet enabled in BTCPay server bitcoin node.

Apart from being "cool" to have, is there a real use case to enable this?

**Comment by britttttk** [comment]
@pavlenex hmmm good question.. I think either mention it in one of the wallet docs... or just add it to FAQ (since steps are fairly short) or maybe create a new doc at some point that is like External/Additional Services or something like that.

**Comment by britttttk** [comment]
As @maltokyo pointed out it would be better to at least provide some type of use case or example of what you can do with it, if it were to be it's own doc.

**Comment by pavlenex** [comment]
Agreed with you guys that documenting how to connect may be an overkill, since it's quite straightforward. Perhaps adding a document for these external wallets or adding a paragraph about them would be handy.
Still haven't had the time to explore fully noded `fully`!

I'm sure @Fonta1n3 can elaborate on current and potential use-cases that we can document.

**Comment by Fonta1n3** [comment]
Hey guys! Thanks for adding Fully Noded connection :) it is an honor!

Asking what the point of Fully Noded is is kind of like asking what the point of bitcoin is. Literally anything you would do with bitcoin you can do with your node via Fully Noded (almost). I am not here to tell you how to use bitcoin but for me personally I like to use the app and my node to import my cold storage xpubs and then I can easily build unsigned transactions and pass them to my offline node for signing. Fully Noded brings BIP39 compatibility to your node in a user friendly way. You can easily pair your hardware wallet with it. HD multisig is the best functionality the app has imho. There is a lot going on in the background that makes it much easier for users to utilize hd multisig, create the multisig, import, generate addresses deterministically and build unsigned transactions and sign them. 

Coin control is another great use case. Go to “transactions” > “utxo’s” and you'll see you have full control over your utxo's. You can lock them, unlock them, consolidate them, and spend from specific utxo’s.

You can prove you own certain funds by signing messages and verify messages from others. You can see your btc balance in fiat by tapping the balance, see your node stats. You can tap an unconfirmed transaction to bump the fee. Use fully noded to optimize the fee rate of all your transactions and bump them if needed.

You can build a raw transaction and verify it before broadcasting.

You can do manual coinjoins using the psbt flow.

Basically i need to make a million tutorials and don't have the time yet! Have tried to make the app as intuitive and self explanatory as possible though. From a practical point of view most people just hodl, so use it to receive to your hodl stash and monitor it privately over tor, if one day you need to spend from it fully noded helps with that too ;) Another very useful feature is the wallet encryption.
